packages/fetchai/skills/generic_seller/strategy.py

```diff
@@ -34,8 +34,8 @@
 DEFAULT_IS_LEDGER_TX = True
 DEFAULT_DATA_MODEL_NAME = "location"
 DEFAULT_DATA_MODEL = {
-    "attribute_one": {"name": "country", "type": "str", "is_required": "True"},
-    "attribute_two": {"name": "city", "type": "str", "is_required": "True"},
```

Comment:
`"True"` was forwarded into the `Attribute` class constructor, which instead expects `bool`